test/base/ViewValidator.scala

```diff
@@ -48,10 +52,13 @@ trait ViewValidator extends MustMatchers {
     elements.first()
   }
 
+  @deprecated("Please use 'page.getElementsBySelector must ...'")
   def getElementsByCss(html: Html, selector: String): Elements = asDocument(html).select(selector)
 
+  @deprecated("Please use 'page.getElementsBySelector must ...'")
   def getElementsByCss(page: String, selector: String): Elements = asDocument(page).select(selector)
```

Comment:
Hmm, I think about not using anything based on the Css, just id.

but still `page.getElementsBySelector` is 100% better than `asDocument(page).select(selector)`